master.cfg

```diff
@@ -0,0 +1,285 @@
+from buildbot.steps.source import Git
+import config_helper
+
+config = config_helper.LoadConfig('build.config')
+slaves = config_helper.SlaveStore(config['SLAVES'])
+
+# This is the dictionary that the buildmaster pays attention to. We also use
+# a shorter alias to save typing.
+c = BuildmasterConfig = {}
+
+####### PROJECT IDENTITY
+
+c['title'] = config['TITLE']
+c['titleURL'] = config['URL']
+c['buildbotURL'] = config['BUILDBOTURL']
+
+####### CHANGESOURCES
+
+# The 'change_source' setting tells the buildmaster how it should find out
+# about source code changes.
+
+c['change_source'] = [
+    GitPoller(("%s%s" % (config['BASEURL'], config['REPOSTUB'])),
+              workdir='gitpoller-workdir',
+              branch='master',
+              pollinterval=300),
+]
+
+####### BUILDSLAVES
+
+c['slavePortnum'] = config['SLAVE_PORT']
+
+# The 'slaves' list defines the set of recognized buildslaves. Each element is
+# a BuildSlave object, specifying a unique slave name and password.
+c['slaves'] = [BuildSlave(s.name(), s.password()) for s in slaves.GetSlaves()]
+
+####### BUILDERS
+
+# The 'builders' list defines the Builders, which tell Buildbot how to perform
+# a build: what steps, and which slaves can execute them.  Note that any
+# particular build will only take place on one slave.
+
+# The main factory, to do fetch via git, autoreconf, configure, make & make
+# check.
+factory = BuildFactory()
+# check out the source
+factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+factory.addStep(ShellCommand(
+    command=["qmake"],
+    name="qmake",
+    haltOnFailure=True,
+    flunkOnFailure=True,
+))
+factory.addStep(Compile(
+    name='make',
+    description='building',
+    descriptionDone='build',
+    command=['make'],
+))
+factory.addStep(Compile(
+    name='unit tests',
+    description='testing',
+    descriptionDone='test',
+    command=['./unittest.sh'],
+    logfiles=config['LOGFILES'],
+    lazylogfiles=True,
+))
+
+# cpplint factory
+cpp_lint_command = (
+    'cpplint.py %s $(find ./ -name "*.h" -or -name "*.cpp" | xargs)' %
+    config['CPP_LINT_ARGS'])
+cpp_lint_factory = BuildFactory()
+cpp_lint_factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+cpp_lint_factory.addStep(ShellCommand(
+   command=cpp_lint_command,
+   haltOnFailure=False,
+   flunkOnFailure=False,
+   warnOnWarnings=True,
+   warnOnFailure=True,
+   description='linting',
+   descriptionDone='lint',
+))
+
+# jslint factory
+js_lint_command = (
+    'gjslint %s -r javascript/ola/' %
+    config['JS_LINT_ARGS'])
+js_lint_factory = BuildFactory()
+js_lint_factory.addStep(Git(repourl=("%s%s" % (config['BASEURL'], config['REPOSTUB']))))
+js_lint_factory.addStep(ShellCommand(
+   command=js_lint_command,
+   haltOnFailure=False,
+   flunkOnFailure=False,
+   warnOnWarnings=True,
+   warnOnFailure=True,
+   description='linting',
+   descriptionDone='lint',
+))
+
+# heap check factory
```

Comment:
Done. I've had to comment this out in master.cfg to not do anything odd in build.config.